Sources/LaunchAgent/LaunchControl.swift

```diff
@@ -182,6 +188,14 @@ extension LaunchControl {
         Process.launchedProcess(launchPath: LaunchControl.launchctl, arguments: arguments)
     }
     
+    /// Run `launchctl bootout` on the agent
+    ///
+    /// Check the status of the job with `.status(_: LaunchAgent)`
+    public func bootout(_ agent: LaunchAgent) throws {        
```

Comment:
It looks like `bootout(_:)` is using the agent's label rather than path, like `unload(_:)` does. If `bootout` doesn't need to use the full path then I would remove `throws` and the test `LaunchControlTests.testBootoutError()`.